development/build/scripts.js

```diff
@@ -370,16 +370,20 @@ const postProcessServiceWorker = (
   mv3BrowserPlatforms,
   fileList,
   applyLavaMoat,
+  testing,
 ) => {
   mv3BrowserPlatforms.forEach((browser) => {
     const appInitFile = `./dist/${browser}/app-init.js`;
     const fileContent = readFileSync('./app/scripts/app-init.js', 'utf8');
-    const fileOutput = fileContent
-      .replace('/** FILE NAMES */', fileList)
-      .replace(
+    let fileOutput = fileContent.replace('/** FILE NAMES */', fileList);
+    if (!testing) {
+      // Lavamoat is always set to true in testing mode
```

Comment:
> Lavamoat is always set to true in testing mode

if this is true we shouldnt need to branch on `testing`

Reply:
This is fixed in following PR: https://github.com/MetaMask/metamask-extension/pull/15157/files#diff-a05d1f077953d24c0d8ddecf64163da2653ba3b728e1ba1f18af6d770767b127R379

Reply:
I think this comment is confusing, I changes it here: https://github.com/MetaMask/metamask-extension/pull/15157/files#diff-a05d1f077953d24c0d8ddecf64163da2653ba3b728e1ba1f18af6d770767b127R381